style.css

```diff
@@ -0,0 +1,278 @@
+    font-size: var(--title-size)
+}
+
+.info-section p {
+    font-size: 12px;
+    max-width: 500px;
+}
+
+.carousel-section {
+    width: 100%;
+    min-height: 220px;
+    display: flex;
+    padding: 15px;
+    background-color: var(--light-blue);
+}
+
+.carousel-container {
+    max-width: 1200px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    margin: 0 auto;
+}
+
+.carousel-display {
+    width: 75%;
+    max-width: 500px;
+    display: flex;
+    background-image: url('https://images.unsplash.com/photo-1532298229144-0ec0c57515c7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1122&q=80');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.prev, .next {
+    min-width: 40px;
+    background: none;
+    border: none;
+    margin: 0 8px 0 8px;
+    font-size: 35px;
+}
+
+.categories-section {
+    width: 100%;
+    max-width: 1200px;
+    display: flex;
+    justify-content: center;
+    gap: 20px;
+    flex-wrap: wrap;
+    margin: 0 auto;
+    margin-top: 20px;
+    padding: 0 20px 0 20px;
+}
+
+.category {
+    width: 20%;
+    min-width: 90px;
+    height: 35px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 500;
+    font-size: 12px;
+    color: var(--dark-blue);
+    border: 1px solid var(--main-blue);
+    border-radius: 5px;
+}
+
+footer {
+    width: 100%;
+    height: 80px;
+    background-color: var(--light-blue);
+    padding: 0 20px 0 20px;
+}
+
+.footer-container {
+    max-width: 1200px;
+    height: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 auto;
+}
+
+footer .social {
+    min-width: 130px;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+footer .social a img {
+    width: 25px;
+}
+
+.copyright {
+    font-size: 10px;
+}
+
+@media (min-width: 500px) {
+    .info-section h1 {
+        font-size: 25px;
+    }
+
+    .carousel-section {
+        min-height: 320px;
+    }
+}
+
+@media (min-width: 768px) {
+    .carousel-section {
+        min-height: 400px;
+    }
+
+    .info-section {
+        min-height: 120px;
+    }
+}
+
+@media (min-width: 1366px) {
+    .carousel-section {
+        min-height: 500px;
+    }
+
+    .carousel-display {
+        max-width: 600px;
+        margin-top: 30px;
+    }
+
+    .info-section {
+        min-height: 140px;
+    }
+
+    .info-section p {
+        font-size: 14px;
+    }
+}
+
+/* Inner page */
+
+.items-section {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 10px;
+    margin-top: 20px;
+}
+
+.items-section h2 {
+    font-size: 28px;
+}
+
+#filter {
+    width: 90%;
+    height: 25px;
+    max-width: 600px;
+}
+
+.items-container {
+    width: 100%;
+    max-width: 1200px;
+    display: flex;
+    flex-wrap: wrap;
```

Comment:
Consider using `display: grid` together with `grid-template-columns` instead of `flex-wrap`